src/lib/secondsToDays.js

```diff
@@ -0,0 +1,6 @@
+const secondsToDays = (seconds)  => {
+  const days = Math.floor(Number(seconds) / (3600*24));
```

Comment:
Something to consider: by flooring the value here you lose some fidelity in the calculations. An alternative would be to keep the raw value in the database and floor or round only when you display the value to the user.